ci-operator/jobs/openshift/private-ci-testing/openshift-private-ci-testing-master-presubmits.yaml

```diff
@@ -41,17 +41,17 @@ presubmits:
           requests:
             cpu: 10m
         volumeMounts:
-        - mountPath: /usr/local/github-ssh-credentials-openshift-bot
-          name: github-ssh-credentials-openshift-bot
+        - mountPath: /usr/local/github-credentials
+          name: github-credentials-openshift-ci-robot-private-git-cloner
```

Comment:
so has there some change in the ay authentication is done in the ci operator ? (just out of curiosity)

Reply:
Yup, these jobs configs are auto-generated using `registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest` container (locally) and we've to include part of this PR (there is check in this PR's CI which verifies the files diff).

Reply:
Rebased, the changes are in master branch already.